tests/rptest/tests/data_transforms_test.py

```diff
@@ -119,3 +122,44 @@ def test_lifecycle(self):
         """
         self._deploy_wasm("identity-xform")
         self._delete_wasm("identity-xform")
+
+    def _produce_input_topic(self) -> TransformVerifierProduceStatus:
+        input_topic = self.topics[1]
+
+        status = TransformVerifierService.oneshot(
+            context=self.test_context,
+            redpanda=self.redpanda,
+            config=TransformVerifierProduceConfig(
+                bytes_per_second='1MB',
+                max_batch_size='512KB',
+                max_bytes='10MB',
+                message_size='1KB',
+                topic=input_topic.name,
+            ))
+        return typing.cast(TransformVerifierProduceStatus, status)
```

Comment:
🔥

Reply:
Maybe it's better to have a `TransformVerifierService` subclass that has the right types for produce/consume instead of the union types... Eh